### PR TITLE
[Team 5] Final Tweaks

### PR DESCRIPTION
**Pull request by Rhys-J**
# Summary

Slight changes to calculations for (hopefully) final PR of team5 agent

## Additional Information

May all agents prosper together

## Test Plan

Wouldn't want to give away our secrets now would we ;)

### CODE REVIEW

**Review by JaafarRammal — DISMISSED** [review]
Code LGTM

**Review by SaraFFdez — DISMISSED** [review]
code LGTM although some variable names are not easy to understand so I can't follow the logic. But if it works as you intend, nice

File: pkg/agents/team5/treaty.go [hunk 1]
```diff
@@ -117,10 +117,10 @@ func (a *CustomAgent5) HandleProposeTreaty(msg messages.ProposeTreatyMessage) {
 	case (treaty.ConditionOp() == messages.GE || treaty.ConditionOp() == messages.GT) && treaty.Condition() == messages.Floor:
 		// Don't agree to treaty which is bounded by you being lower down in the tower for same as reasons above
 		fallthrough
-	case treaty.Condition() == messages.HP && treaty.ConditionValue() < a.HealthInfo().WeakLevel*2:
+	case treaty.Condition() == messages.HP && treaty.ConditionValue() < a.HealthInfo().WeakLevel:
 		//Reject any HP condition based treaty if the condition is too strict, and you would be put into critical by not eating
 		fallthrough
-	case treaty.Condition() == messages.Floor && treaty.ConditionValue() != 1 && treaty.Duration() >= a.HealthInfo().MaxDayCritical:
+	case treaty.Condition() == messages.Floor:
 		//Reject any floor condition that involves us not being on the top floor and lasts for more than days you can survive in critical
```
Comment (hussain2603) [inline pkg/agents/team5/treaty.go:124]: this comment should be changed now, right?

File: pkg/agents/team5/treaty.go [hunk 2]
```diff
@@ -135,7 +135,7 @@ func (a *CustomAgent5) HandleProposeTreaty(msg messages.ProposeTreatyMessage) {
 
 	//TODO: Develop the decision calculation.
```
Comment (hussain2603) [inline pkg/agents/team5/treaty.go:136]: is this still a TODO?

**Review by hussain2603 — CHANGES_REQUESTED** [review]
can also get rid of those comments in dayPassed() if you wanted to. otherwise, nice work :).

**Review by SaraFFdez — APPROVED** [review]

**Review by hussain2603 — APPROVED** [review]
looks good thanks xx 😍

**Pull request merged**